iOS Example Frame/Podfile.lock

```diff
@@ -1,9 +1,9 @@
 PODS:
-  - Checkout (4.0.0):
+  - Checkout (4.0.1):
```

Comment:
@ehab-al-cko  I dont see any changes to pod file itself. I was expecting the 4.0.1 should have been on the pod file itself.

Reply:
The pod uses a generic dependency to Frames, which will just update to latest
```
  pod 'Frames', :path => '../'
```